recipes/pymatgen/meta.yaml

```diff
@@ -0,0 +1,132 @@
+{% set name = "pymatgen" %}
+{% set version = "2019.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2872f431cb1e9dd64e52f89ec669050ea7e90d2acdfcad4d1fc25b1e1d6e8b19
+
+build:
+  skip: true  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  entry_points:
+    - pmg = pymatgen.cli.pmg:main
+    - feff_input_generation = pymatgen.cli.feff_input_generation:main
+    - feff_plot_cross_section = pymatgen.cli.feff_plot_cross_section:main
+    - feff_plot_dos = pymatgen.cli.feff_plot_dos:main
+    - gaussian_analyzer = pymatgen.cli.gaussian_analyzer:main
+    - get_environment = pymatgen.cli.get_environment:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - setuptools >=18.0
+  host:
+    - python
+    - pip
+    - cython
+    - numpy >=1.14.3
+    - requests
+    - ruamel
+    - ruamel.yaml >=0.15.6
+    - monty >=1.0.6
+    - scipy >=1.0.1
+    - pydispatcher >=2.0.5
+    - tabulate
+    - spglib >=1.9.9.44
+    - networkx >=2.2
+    - matplotlib >=1.5
+    - palettable >=3.1.1
+    - sympy
+    - pandas
+    - pybtex
+    - apscheduler >=2.1.0
+    - netcdf4
+    - ase >=3.3
+    - vtk >=6.0.0
+    - dataclasses >=0.6  # [py<37]
+  run:
+    - python
+    - cython
```

Comment:
Is cython a run time requirement?

Reply:
You're right, I guess I messed this up during debugging.